### DISCUSSION TITLE
Trans-Ancestry Quality Control

### DISCUSSION THREAD

**Discussion by TravisMizeIGH** [PGScatalog/pgsc_calc#305] [Q&A]
Hello,

I am attempting to run PGSC with a trans-ancestry sample and was hoping you could clarify a few things. 

1) Should the input sample first undergo ancestry-specific quality control (e.g., maf, geno, mind, etc.) when adjusting PGS for genetic ancestry? This can create large genotype missingness across superpopulations due to LD structure when cohorts are combined into a single file. As an example, my cohorts (AFR, EAS, EUR, HIS, and SAS) have genotyping rate > 0.99 separately, but ~0.55 when combined. How does PGSC handle low genotyping rate in this instance and would this cause any issues I should be aware of?

2) Would it be possible to add to your preparation page (https://pgsc-calc.readthedocs.io/en/latest/how-to/prepare.html) what QC should be performed, based on the analyses being conducted? Something similar to https://choishingwan.github.io/PRS-Tutorial/plink/, perhaps?

**Comment by smlmbrt** [comment]
Hi @TravisMizeIGH, I'll answer both questions:

1. I think this is currently up to the user to test, it will probably depend on the dataset and the use-case. The current pipeline in `--run_ancestry` mode will use the allele frequency in the reference panel to impute the missing allele dosage during PGS Calculation, in the normal mode it will use the allele frequency in your samples to impute the dosage (and also calculate the AVG PGS for non-missing sites). MAF and missingness in your target samples are currently accounted for in the variant-selection step for PCA and we can implement filters for whether these sites should be included in score calculation if users thought this type of QC was more useful in the pipeline vs. pre-pipeline. 
2. This is an excellent suggestion, I will definitely add documentation around this.
